nova/templates/configmaps/nova.conf.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  nova.conf: |+
+    [DEFAULT]
+    debug = {{ misc.debug }}
+    vif_plugging_is_fatal = False
```

Comment:
Thank you. I fixed that one and verified manually that all other FIXME were there.